HISTORY.md

```diff
@@ -2,6 +2,44 @@
 title: Release History
 ---
 
+# 2.2.0 (2024-01-08)
+
+# Highlights
+This release incorporates several majors changes to Mesa. Notably, this provides two experimental features. (1) It reimagines Mesa's agent handling to have agents in the model class and store them in sets. The intent is to make Mesa more efficient and give users more control over agent scheduling to allow for innumerable options. (2) It adds property layer and property grid so users can layer on multiple grids making Mesa and Geo-Mesa more similar.
```

Comment:
Needs to link to the feedbacks: https://github.com/projectmesa/mesa/discussions/1919 and https://github.com/projectmesa/mesa/discussions/1932.

Reply:
I amended the commit to include this.